lix/source-code/client/src/browser-auth.ts

```diff
@@ -95,4 +119,4 @@ export async function getUser() {
 	}
 }
 
-export default { login, logout, getUser }
+export default { login, logout, getUser, addPermissions }
```

Comment:
why export default? in that moment you break tree-shaking. 

i recommend to export each function that should be exported. consumers can then decide what they want to consume, or use a namespace import `import * as auth` to achieve the default export like imports without loosing tree-shaking.

Reply:
In that case, I recommend to export a named object.

```ts
export const browserAuth: AuthObject = {
  login, 
  // ...
}

```

**Pros**

- you can define an `AuthObject` type which must be followed (your default export approach doesn't allow that)

**Cons**

N/A

Reply:
Btw if you resolve the conversation, I get no notification to respond